TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Models/DqtNote.cs

```diff
@@ -0,0 +1,12 @@
+namespace TeachingRecordSystem.Core.DataStore.Postgres.Models;
+
+public class DqtNote
+{
+    public required Guid Id { get; set; }
+    public required Guid PersonId { get; set; }
+    public required string NoteText { get; set; }
+    public required DateTime ModifiedOn { get; set; }
```

Comment:
To match other models
```suggestion
    public required DateTime UpdatedOn { get; set; }
```